app/src/main/java/org/jellyfin/mobile/utils/Constants.kt

```diff
@@ -12,7 +12,7 @@ object Constants {
 
     // Webapp constants
     const val MINIMUM_WEB_VIEW_VERSION = 80
-    const val WEB_CONFIG_PATH = "config.json"
+    val MAIN_BUNDLE_PATH_REGEX = Regex(""".*/main.[a-f\d]+.bundle.js""")
```

Comment:
How can we be sure the hash of the bundle never changes? If webpack was updated and they decided to use sha sums instead of md4 it might break.

Reply:
It shouldn't, the default digest is hex, regardless of the hashing algorithm.

Reply:
Ok sha hashes was a wrong example, but my point still stands

Reply:
I'd say it's highly unlikely for Jellyfin Web or Webpack to switch to a different digest - the web app will be EOL before that..

Reply:
Anyway, nothing is future-proof, if I already happened at least 2 times with v10.6 and v10.7, I don't doubt that it will change again for v10.8, unless the native client appears before, we have the webview hybrid client for a while...